static/app/views/performance/transactionSummary/transactionTags/content.tsx

```diff
@@ -175,13 +196,28 @@ const InnerContent = (
             <EnvironmentPageFilter />
             <DatePageFilter alignDropdown="left" />
           </PageFilterBar>
-          <SearchBar
+          <StyledSearchBar
             organization={organization}
             projectIds={eventView.project}
             query={query}
             fields={eventView.fields}
             onSearch={handleSearch}
           />
+          <CompactSelect
+            value={aggregateColumn}
+            options={X_AXIS_SELECT_OPTIONS}
+            onChange={opt => {
+              trackAdvancedAnalyticsEvent(
+                'performance_views.tags.change_aggregate_column',
+                {
+                  organization,
+                  value: opt.value,
+                }
+              );
```

Comment:
Wasn't sure whether to copypasta `trackAnalyticsEvent` or `trackAdvancedAnalyticsEvent` but there were more of the latter in the codebase

Reply:
That's the right one, `trackAnalyticsEvents` should be showing as deprecated if your editor supports such things (it has `@deprecated` in the comments for it.)